websites/Y/yabancıdizi/metadata.json

```diff
@@ -12,7 +12,7 @@
 		"ga_IE": "Tá yabancıdizi ar cheann de na soláthraithe seó teilifíse eachtrach Tuircis is mó ar an idirlíon.",
 		"nl": "yabancıdizi is een van de grootste Turkse buitenlandse tv-showproviders op internet."
 	},
-	"url": "yabancidizi.pro",
+	"url": "yabancidizi.co",
 	"version": "1.1.19",
```

Comment:
Bump version
```suggestion
	"version": "1.1.20",
```

Reply:
Should be breaking changes if the domain was changed